frontend/src/pages/talks/TalkSingleEntry.tsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+
+import { Box, Paper, Typography } from '@mui/material';
+import { TalkEntry } from 'src/services/mocks';
+import { extractVideoId } from 'src/utils/video';
+import { useTranslation } from 'react-i18next';
+
+const TalkSingleEntry = ({ talk }: { talk: TalkEntry; compact?: boolean }) => {
```

Comment:
The `compact` prop doesn't seem to be used here.

Should we remove it?

Reply:
Yes, It's removed.